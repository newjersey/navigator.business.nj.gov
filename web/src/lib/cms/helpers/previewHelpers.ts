/* eslint-disable @typescript-eslint/no-explicit-any */

export type PreviewProps = {
  entry?: any;
  window: Window;
  document: Document;
  fieldsMetaData: any;
  widgetsFor: (string: string) => any;
  widgetFor: (string: string) => any;
  getAsset: (string: string) => any;
};
