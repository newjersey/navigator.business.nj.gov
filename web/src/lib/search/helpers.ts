import { LabelledContent, LabelledContentList, Match } from "@/lib/search/typesForSearch";

export const findMatchInBlock = (blockTexts: string[], term: string, match: Match): Match => {
  for (const blockText of blockTexts) {
    if (blockText.includes(term)) {
      const index = blockText.indexOf(term);
      const startIndex = index - 50 < 0 ? 0 : index - 50;
      const endIndex = term.length + index + 50;
      match.snippets.push(blockText.slice(startIndex, endIndex));
    }
  }
  return match;
};

export const findMatchInLabelledText = (
  labelledTexts: LabelledContent[],
  term: string,
  match: Match
): Match => {
  for (const labelledText of labelledTexts) {
    if (labelledText.content?.includes(term)) {
      match.snippets.push(`${labelledText.label}: ${labelledText.content}`);
    }
  }
  return match;
};

export const findMatchInListText = (listTexts: LabelledContentList[], term: string, match: Match): Match => {
  for (const listText of listTexts) {
    for (const item of listText.content) {
      if (item?.includes(term)) {
        match.snippets.push(`${listText.label}: ${item}`);
      }
    }
  }
  return match;
};
