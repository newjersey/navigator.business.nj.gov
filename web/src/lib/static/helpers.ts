import { convertTaskMd } from "@/lib/utils/markdownReader";
import fs from "fs";
import path from "path";

export const loadUrlSlugByFilename = (
  fileName: string,
  filingsDir: string
): string => {
  const fullPath = path.join(filingsDir, `${fileName}`);
  const fileContents = fs.readFileSync(fullPath, "utf8");
  const content = convertTaskMd(fileContents);
  return content.urlSlug;
};

export const getFileNameByUrlSlug = (
  currPath: string,
  urlSlug: string
): string => {
  const fileNames = fs.readdirSync(currPath);
  const matchingFileName = fileNames.find((fileName) => {
    return urlSlug === loadUrlSlugByFilename(fileName, currPath);
  });
  if (!matchingFileName) {
    throw `Task with urlSlug ${urlSlug} not found`;
  }

  return matchingFileName;
};
