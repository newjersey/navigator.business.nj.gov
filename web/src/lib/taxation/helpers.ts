import { postTaxFilingsOnboarding } from "@/lib/api-client/apiClient";
import { UpdateQueue } from "@/lib/UpdateQueue";
import { LookupLegalStructureById } from "@businessnjgovnavigator/shared/legalStructure";
import { ProfileData } from "@businessnjgovnavigator/shared/profileData";

const displayBusinessName = (profileData: ProfileData): boolean => {
  const hasBusinessName = LookupLegalStructureById(
    profileData.legalStructureId,
  ).elementsToDisplay.has("businessName");
  const notEmptyString = profileData.businessName !== "";
  return hasBusinessName && notEmptyString;
};

const displayResponsibleOwnerName = (profileData: ProfileData): boolean => {
  const hasResponsibleOwnerName = LookupLegalStructureById(
    profileData.legalStructureId,
  ).elementsToDisplay.has("responsibleOwnerName");
  const notEmptyString = profileData.responsibleOwnerName !== "";
  return hasResponsibleOwnerName && notEmptyString;
};

interface Props {
  updateQueue: UpdateQueue;
  stagedProfileData?: ProfileData;
}

export const gov2GovTaxFiling = async (props: Props): Promise<void> => {
  const currentBusiness = props.updateQueue.currentBusiness();

  const updateQueue = props.updateQueue;
  const profileData = props.stagedProfileData || currentBusiness.profileData;
  const displayBusinessNameValue = displayBusinessName(profileData);
  const displayResponsibleOwnerNameValue = displayResponsibleOwnerName(profileData);

  if (!displayBusinessNameValue && !displayResponsibleOwnerNameValue) return;
  if (!profileData.taxId) return;

  const encryptedTaxIdToSubmitToTaxApi =
    profileData.taxId === currentBusiness.profileData.taxId && profileData.encryptedTaxId
      ? profileData.encryptedTaxId
      : "";

  const businessNameToSubmitToTaxApi = displayBusinessNameValue
    ? profileData.businessName
    : profileData.responsibleOwnerName;

  const userDataToSet = await postTaxFilingsOnboarding({
    taxId: profileData.taxId,
    businessName: businessNameToSubmitToTaxApi,
    encryptedTaxId: encryptedTaxIdToSubmitToTaxApi,
  });

  updateQueue.queue(userDataToSet).queueProfileData({
    taxId: profileData.taxId,
    encryptedTaxId: encryptedTaxIdToSubmitToTaxApi,
  });
};
