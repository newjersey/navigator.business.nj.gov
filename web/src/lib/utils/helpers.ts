import { ConfigType, getMergedConfig } from "@/contexts/configContext";
import { FlowType, OnboardingStatus } from "@/lib/types/types";
import {
  BusinessPersona,
  Municipality,
  MunicipalityDetail,
  ProfileData,
  UserData,
} from "@businessnjgovnavigator/shared";
import { useEffect, useRef } from "react";

export const useMountEffect = (fun: () => void): void => {
  // eslint-disable-next-line react-hooks/exhaustive-deps
  return useEffect(fun, []);
};

export const groupBy = <T>(array: T[], predicate: (value: T, index: number, array: T[]) => string): T[][] =>
  Object.values(
    array.reduce((acc, value, index, array) => {
      (acc[predicate(value, index, array)] ||= []).push(value);
      return acc;
    }, {} as { [key: string]: T[] })
  );

export const useOnWindowResize = (fun: () => void): void => {
  return useEffect(() => {
    window.addEventListener("resize", fun);
    return function cleanup() {
      window.removeEventListener("resize", fun);
    };
  });
};

export const useMountEffectWhenDefined = (fun: () => void, thingToBeDefined: unknown | undefined): void => {
  const effectOccurred = useRef<boolean>(false);
  useEffect(() => {
    if (thingToBeDefined && !effectOccurred.current) {
      effectOccurred.current = true;
      fun();
    }
  }, [thingToBeDefined, fun]);
};

export const onEscape = (e: KeyboardEvent, handler: () => void): void => {
  if (e.key === "Escape") {
    e.preventDefault();
    handler();
  }
};

export const templateEval = (template: string, args: Record<string, string>): string => {
  let newTemplate = template;
  for (const key of Object.keys(args)) {
    const pattern = `\\\${${key}}`;
    newTemplate = newTemplate.replace(new RegExp(pattern, "g"), args[key]);
  }
  return newTemplate;
};

export const templateEvalWithExtraSpaceRemoval = (template: string, args: Record<string, string>): string => {
  let newTemplate = template;
  for (const key of Object.keys(args)) {
    const pattern = `\\\${${key}} `;
    newTemplate = newTemplate.replace(new RegExp(pattern, "g"), args[key]);
  }
  return newTemplate;
};

export const rswitch = <T>(param: string, cases: { default: T; [k: string]: T }): T => {
  return cases[param] ?? cases.default;
};

export const scrollToTop = (props?: { smooth?: boolean }): void => {
  props?.smooth
    ? setTimeout(() => {
        return window.scrollTo({ top: 0, behavior: "smooth" });
      }, 100)
    : window.scrollTo(0, 0);
};

export const scrollToTopOfElement = (
  element: HTMLDivElement | null,
  { isDesktop }: { isDesktop: boolean }
) => {
  let y = 0;
  if (element) {
    y = window.scrollY + element.getBoundingClientRect().top;
    if (!isDesktop) {
      const mobileNavBarHeight = 47;
      y -= mobileNavBarHeight;
    }
  }
  setTimeout(() => {
    return window.scrollTo({ top: y, behavior: "smooth" });
  }, 100);
};

interface AlertProps {
  variant: "success" | "warning" | "error";
  body: string;
  header?: string;
  link?: string;
}

export const OnboardingStatusLookup = (
  configOverrides?: ConfigType
): Record<OnboardingStatus, AlertProps> => {
  const config = configOverrides ?? getMergedConfig();
  return {
    SUCCESS: {
      body: config.profileDefaults.successTextBody,
      header: config.profileDefaults.successTextHeader,
      variant: "success",
    },
    ERROR: {
      body: config.profileDefaults.errorTextBody,
      header: config.profileDefaults.errorTextHeader,
      variant: "error",
    },
  };
};

export const getUserNameOrEmail = (userData: UserData | undefined): string => {
  if (userData?.user.name) {
    return userData.user.name;
  } else if (userData?.user.email) {
    return userData.user.email;
  } else {
    return getMergedConfig().navigationDefaults.myNJAccountText;
  }
};

export const validateEmail = (email: string): boolean => {
  return !!/^(([^\s"(),.:;<>@[\\\]]+(\.[^\s"(),.:;<>@[\\\]]+)*)|(".+"))@((\[(?:\d{1,3}\.){3}\d{1,3}])|(([\dA-Za-z-]+\.)+[A-Za-z]{2,}))$/.test(
    String(email).toLowerCase()
  );
};

export const getPhoneNumberFormat = (phoneNumber: string): string => {
  const length = phoneNumber.length;
  if (length === 0) {
    return phoneNumber;
  }
  if (length < 4) {
    return `(${phoneNumber}`;
  }
  if (length < 7) {
    return `(${phoneNumber.slice(0, 3)}) ${phoneNumber.slice(3)}`;
  }
  return `(${phoneNumber.slice(0, 3)}) ${phoneNumber.slice(3, 6)}-${phoneNumber.slice(6, 10)}`;
};

export const getFlow = (data: UserData | ProfileData): FlowType => {
  const persona: BusinessPersona = isUserData(data) ? data.profileData.businessPersona : data.businessPersona;

  return persona || "STARTING";
};

export function isUserData(data: UserData | ProfileData): data is UserData {
  return (data as UserData).user !== undefined;
}

// eslint-disable-next-line @typescript-eslint/no-explicit-any
export const flattenObject = (obj: any) => {
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  const flattened: any = {};

  if (typeof obj === "object" && obj !== null && !Array.isArray(obj)) {
    for (const key of Object.keys(obj)) {
      const value = obj[key];

      if (typeof value === "object" && value !== null && !Array.isArray(value)) {
        Object.assign(flattened, flattenObject(value));
      } else {
        flattened[key] = value;
      }
    }

    return flattened;
  } else {
    return obj;
  }
};

export const mapMunicipalityDetailToMunicipality = (municipalityDetail: MunicipalityDetail): Municipality => {
  return {
    displayName: municipalityDetail.townDisplayName,
    id: municipalityDetail.id,
    name: municipalityDetail.townName,
    county: municipalityDetail.countyName,
  };
};
