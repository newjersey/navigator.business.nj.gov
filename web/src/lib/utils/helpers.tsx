import { NavDefaults } from "@/display-defaults/NavDefaults";
import { OnboardingDefaults } from "@/display-defaults/onboarding/OnboardingDefaults";
import { ProfileDefaults } from "@/display-defaults/ProfileDefaults";
import {
  OnboardingStatus,
  ProfileError,
  Roadmap,
  SectionCompletion,
  sectionNames,
  SectionType,
  Step,
  Task,
} from "@/lib/types/types";
import { Preferences, UserData } from "@businessnjgovnavigator/shared/";
import { ParsedUrlQuery } from "querystring";
import React, { ReactElement, useEffect, useRef } from "react";

// eslint-disable-next-line react-hooks/exhaustive-deps
export const useMountEffect = (fun: () => void): void => useEffect(fun, []);

export const useOnWindowResize = (fun: () => void): void =>
  useEffect(() => {
    window.addEventListener("resize", fun);
    return function cleanup() {
      window.removeEventListener("resize", fun);
    };
  });

export const useMountEffectWhenDefined = (fun: () => void, thingToBeDefined: unknown | undefined): void => {
  const effectOccurred = useRef<boolean>(false);
  useEffect(() => {
    if (thingToBeDefined && !effectOccurred.current) {
      effectOccurred.current = true;
      fun();
    }
  }, [thingToBeDefined, fun]);
};

export const onEscape = (e: KeyboardEvent, handler: () => void): void => {
  if (e.key === "Escape") {
    e.preventDefault();
    handler();
  }
};

export const templateEval = (template: string, args: Record<string, string>): string =>
  template.replace(/\${(\w+)}/g, (_, v) => args[v]);

export const getTaskFromRoadmap = (roadmap: Roadmap | undefined, taskId: string): Task | undefined =>
  stepInRoadmap(roadmap, taskId)?.tasks.find((task) => task.id === taskId);

export const getSectionCompletion = (
  roadmap: Roadmap | undefined,
  userData: UserData | undefined
): SectionCompletion => {
  if (!roadmap || !userData) {
    return {} as SectionCompletion;
  }
  const taskMap = sectionsToTasksMap(roadmap) as Record<SectionType, Task[]>;
  return sectionNames.reduce((accumulator, currentValue: SectionType) => {
    accumulator[currentValue] =
      taskMap[currentValue]?.every((task: Task) => userData.taskProgress[task.id] === "COMPLETED") ?? false;
    return accumulator;
  }, {} as SectionCompletion);
};
interface SectionPosition {
  current: SectionType;
  next: SectionType | undefined;
}

export const getSectionPositions = (
  sectionCompletion: SectionCompletion,
  roadmap: Roadmap,
  taskId: string
): SectionPosition => {
  const currentSection = stepInRoadmap(roadmap, taskId)?.section as SectionType;
  const nextSection = sectionNames
    .slice(sectionNames.indexOf(currentSection))
    .find((currentValue: SectionType) => !sectionCompletion[currentValue]);
  return { current: currentSection, next: nextSection };
};

export const setPreferencesCloseSection = (preferences: Preferences, current: SectionType): Preferences => {
  return {
    ...preferences,
    roadmapOpenSections: preferences.roadmapOpenSections.filter((currentValue) => currentValue !== current),
  } as Preferences;
};

export const getModifiedTaskContent = (
  roadmap: Roadmap | undefined,
  task: Task,
  field: Exclude<keyof Task, "unlockedBy" | "required">
): string => {
  const taskInRoadmap = getTaskFromRoadmap(roadmap, task.id);
  if (taskInRoadmap && taskInRoadmap[field] !== task[field]) {
    return taskInRoadmap[field] || "";
  }
  return task[field] || "";
};

export const rswitch = <T,>(param: string, cases: { default: T; [k: string]: T }): T => {
  if (cases[param]) {
    return cases[param];
  } else {
    return cases.default;
  }
};

export const scrollToTop = (): void => {
  window.scrollTo(0, 0);
};

export const isStepCompleted = (step: Step, userData: UserData | undefined): boolean => {
  return step.tasks.every((it) => {
    const taskProgress = (userData?.taskProgress && userData.taskProgress[it.id]) || "NOT_STARTED";
    return taskProgress === "COMPLETED";
  });
};

interface AlertProps {
  variant: "success" | "warning" | "error";
  body: string;
  header?: string;
  link?: string;
}

export const OnboardingStatusLookup: Record<OnboardingStatus, AlertProps> = {
  SUCCESS: {
    body: ProfileDefaults.successTextBody,
    header: ProfileDefaults.successTextHeader,
    variant: "success",
  },
  ERROR: {
    body: ProfileDefaults.errorTextBody,
    header: ProfileDefaults.errorTextHeader,
    variant: "error",
  },
};

export const OnboardingErrorLookup: Record<ProfileError, string> = {
  REQUIRED_LEGAL: OnboardingDefaults.errorTextRequiredLegal,
  REQUIRED_EXISTING_BUSINESS: OnboardingDefaults.errorTextRequiredExistingBusiness,
};

export const getUserNameOrEmail = (userData: UserData | undefined): string => {
  if (userData?.user.name) return userData.user.name;
  else if (userData?.user.email) return userData.user.email;
  else return NavDefaults.myNJAccountText;
};

export const validateEmail = (email: string): boolean => {
  return !!String(email)
    .toLowerCase()
    .match(
      /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
    );
};

export const getUrlSlugs = (roadmap: Roadmap | undefined): string[] => {
  if (!roadmap) return [];
  const { steps } = roadmap;
  return steps.reduce((acc: string[], currStep: Step) => {
    const { tasks } = currStep;
    return [...acc, ...tasks.map((task) => task.urlSlug)];
  }, []);
};

export const setHeaderRole = (
  ariaLevel: number,
  classProperties?: string
): (({ children }: { children: string[] }) => ReactElement) => {
  const createElement = ({ children }: { children: string[] }): ReactElement => {
    return (
      <div role="heading" aria-level={ariaLevel} className={classProperties ? classProperties : ""}>
        {children}
      </div>
    );
  };

  return createElement;
};

export const getSectionNames = (roadmap: Roadmap | undefined): SectionType[] => {
  if (!roadmap) return [];
  const { steps } = roadmap;
  const sections: SectionType[] = [];
  steps.forEach((step) => {
    sections.push(step.section);
  });
  return [...new Set(sections)];
};

export const createRoadmapSections = (
  roadmapSections: SectionType[],
  userData: UserData | undefined,
  getSection: (sectionName: SectionType, openStatus: boolean | undefined) => JSX.Element
): JSX.Element[] => {
  return roadmapSections.map((sectionName) => {
    const openStatus = userData?.preferences.roadmapOpenSections.includes(sectionName);
    return getSection(sectionName, openStatus);
  });
};

export const camelCaseToSentence = (text: string): string => {
  const spacedCase = text
    .split(/(?=[A-Z])/)
    .join(" ")
    .toLowerCase();
  return spacedCase.charAt(0).toUpperCase() + spacedCase.slice(1);
};
const sectionsToTasksMap = (roadmap: Roadmap | undefined): Record<SectionType, Task[]> | undefined =>
  roadmap?.steps.reduce((accumulator, currentValue: Step) => {
    accumulator[currentValue.section] = [...(accumulator[currentValue.section] || []), ...currentValue.tasks];
    return accumulator;
  }, {} as Record<SectionType, Task[]>);

const stepInRoadmap = (roadmap: Roadmap | undefined, taskId: string): Step | undefined =>
  roadmap?.steps.find((step) => step.tasks.find((task) => task.id === taskId));

export const splitAndBoldSearchText = (displayText: string, searchText: string): ReactElement => {
  const index = displayText.toLowerCase().indexOf(searchText.toLowerCase());
  if (index >= 0) {
    const prefixText = displayText.substr(0, index);
    const toBold = displayText.substr(index, searchText.length);
    const afterText = displayText.substr(index + searchText.length);
    return (
      <span style={{ whiteSpace: "pre-wrap" }}>
        {prefixText}
        <span className="text-bold">{toBold}</span>
        {afterText}
      </span>
    );
  } else {
    return <>{displayText}</>;
  }
};

export const getDollarValue = (currVal: string | number, multiplyBy = 1): string => {
  return new Intl.NumberFormat("en-US", { style: "currency", currency: "USD" }).format(
    parseFloat(currVal.toString()) * multiplyBy
  );
};

export const zipCodeRange = (value: string) => {
  const parsedValue = parseInt(value);
  if (typeof parsedValue !== "number") return false;
  return parsedValue >= 7001 && parsedValue <= 8999;
};

export const featureFlags = (query: ParsedUrlQuery): Record<"featureDisableFormation", boolean> => {
  const featureDisableFormation = query?.formation
    ? query?.formation === "false"
    : process.env.FEATURE_DISABLE_FORMATION
    ? process.env.FEATURE_DISABLE_FORMATION === "true"
    : false;
  return {
    featureDisableFormation,
  };
};

export const getStringifiedAddress = (
  addressLine1: string,
  city: string,
  state: string,
  zipcode: string,
  addressLine2?: string
) => {
  return `${addressLine1}, ${addressLine2 ? `${addressLine2}, ` : ""}${city}, ${state}, ${zipcode}`;
};
