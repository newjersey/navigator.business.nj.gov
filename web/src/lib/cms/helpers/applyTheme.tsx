import { PreviewProps } from "@/lib/cms/helpers/previewHelpers";
import { createTheme, ThemeProvider } from "@mui/material";

// eslint-disable-next-line react/display-name
export const applyTheme = (child: (props: PreviewProps) => JSX.Element) => (props: PreviewProps) =>
  <ThemeProvider theme={createTheme()}>{child(props)}</ThemeProvider>;
