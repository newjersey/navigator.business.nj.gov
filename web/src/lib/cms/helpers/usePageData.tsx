import { PreviewProps } from "@/lib/cms/helpers/previewHelpers";

export const usePageData = <T,>(props: PreviewProps) => {
  const { body, ...data } = JSON.parse(JSON.stringify(props.entry.getIn(["data"])));
  return { contentMd: body, ...data } as T;
};
