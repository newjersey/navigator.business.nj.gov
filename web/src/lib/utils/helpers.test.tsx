import { kabobSnakeSentenceToCamelCase, splitAndBoldSearchText } from "./helpers";

describe("helpers", () => {
  describe("splitAndBoldSearchText", () => {
    it("properly displays the found text as bold", () => {
      expect(splitAndBoldSearchText("food truck", "truck")).toMatchSnapshot();
    });

    it("properly displays the text as not bould when not found", () => {
      expect(splitAndBoldSearchText("food truck", "firm")).toMatchSnapshot();
    });
  });

  describe("kabobSnakeSentenceToCamelCase", () => {
    it("converts kabob-case to camelCase", () => {
      expect(kabobSnakeSentenceToCamelCase("food-truck-whatever")).toEqual("foodTruckWhatever");
    });

    it("converts sentence case to camelCase", () => {
      expect(kabobSnakeSentenceToCamelCase("food truck whatever")).toEqual("foodTruckWhatever");
    });

    it("converts snake_case to camelCase", () => {
      expect(kabobSnakeSentenceToCamelCase("FOOD_TRUCK_WHATEVER")).toEqual("foodTruckWhatever");
    });
  });
});
