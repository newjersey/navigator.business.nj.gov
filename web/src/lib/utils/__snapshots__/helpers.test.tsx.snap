// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`helpers splitAndBoldSearchText properly displays the found text as bold 1`] = `
<span
  style={
    {
      "whiteSpace": "pre-wrap",
    }
  }
>
  food 
  <span
    className="text-bold"
  >
    truck
  </span>
  
</span>
`;

exports[`helpers splitAndBoldSearchText properly displays the text as not bould when not found 1`] = `
<React.Fragment>
  food truck
</React.Fragment>
`;
